scripts/make.sh

```diff
@@ -1,4 +1,4 @@
 #! /usr/bin/env bash
 
 cd src
-luajit ../scripts/pack.lua > ../lulpeg.lua
\ No newline at end of file
+tarantool ../scripts/pack.lua > ../lulpeg.lua
```

Comment:
We don't use make.sh in our builds, so maybe it is better to leave it unchanged. From the other side of view this can be convenient for development. I leave it as the open question.